picoquic/quicctx.c

```diff
@@ -144,19 +144,15 @@ static uint8_t picoquic_cleartext_draft_23_salt[] = {
 
 const picoquic_version_parameters_t picoquic_supported_versions[] = {
     { PICOQUIC_FOURTEENTH_INTEROP_VERSION,
-        picoquic_version_header_17,
         sizeof(picoquic_cleartext_draft_23_salt),
         picoquic_cleartext_draft_23_salt },
     { PICOQUIC_THIRTEENTH_INTEROP_VERSION,
-        picoquic_version_header_17,
         sizeof(picoquic_cleartext_draft_22_salt),
         picoquic_cleartext_draft_22_salt },
     { PICOQUIC_INTERNAL_TEST_VERSION_2,
-        picoquic_version_header_17,
         sizeof(picoquic_cleartext_internal_test_1_salt),
         picoquic_cleartext_internal_test_1_salt },
     { PICOQUIC_INTERNAL_TEST_VERSION_1,
-        picoquic_version_header_17,
         sizeof(picoquic_cleartext_internal_test_1_salt),
         picoquic_cleartext_internal_test_1_salt }
 };
```

Comment:
Yes, "version 17" is the last time the packet code changed, we use only one code now, and we can remove that as you suggest. On the other hand, the packet codes are specified to be version dependent. There are talks of exercising that, as part of protection against "version number ossification". But I guess that we can change the code again when we do have a spec that exercises the variations.